### PR TITLE
feat: configure security scorecard

### PR DESCRIPTION
**Pull request by kenneth-rosario**

### CODE REVIEW

File: .github/workflows/scorecard.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+name: Scorecard supply-chain security
+on:
+  # For Branch-Protection check. Only the default branch is supported. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection
+  branch_protection_rule:
+  # To guarantee Maintained check is occasionally updated. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained
+  schedule:
+    - cron: '0 */12 * * *'
```
Comment (anniefu) [inline .github/workflows/scorecard.yml:9]: Could you add a comment for how often this is? I assume every 12 hours?

Comment (kenneth-rosario) [reply .github/workflows/scorecard.yml]: Yes, added a comment

**Review by anniefu — APPROVED** [review]

File: .github/workflows/scorecard.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    runs-on: ubuntu-latest
+    permissions:
+      # Needed to upload the results to code-scanning dashboard.
+      security-events: write
+      # Needed to publish results and get a badge (see publish_results below).
+      id-token: write
+
+    steps:
+      - name: "Checkout code"
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
```
[the remark this answers is not in the document]
Comment (kenneth-rosario) [reply .github/workflows/scorecard.yml:28]: Synced offline

**Comment by dazuma** [comment]
This was a `chore:`, not a `feat:`, as it has no functional impact on the library and should not have triggered a library release. Please be more careful about conventional commit message types.

**Pull request merged**